client/app/views/steps/password.coffee

```diff
@@ -32,6 +36,29 @@ module.exports = class PasswordView extends StepView
         @$visibilityIcon.attr 'xlink:href', data.visibilityIcon
 
 
+    initialize: (args...) ->
+        super args...
+        # lowest level is 1 to display a red little part
+        @passwordStrength = {percentage: 1, label: 'weak'}
+        @updatePasswordStrength = updatePasswordStrength.bind(@)
+
+
+    updatePasswordStrength= _.throttle( ->
+        password = @$('input[name=password]').val()
```

Comment:
We should use to cache jQuery queries, as every call is pretty expensive. (See http://gregfranko.com/blog/jquery-best-practices/). We can to something like
```coffeescript
@passwordInput ?= @$('input[name=password')
```
and then work only with `@passwordInput`.